server/catgenome/src/main/java/com/epam/catgenome/manager/vcf/VcfManager.java

```diff
@@ -197,9 +191,6 @@ public VcfFile registerVcfFile(FeatureIndexedFileRegistrationRequest request) {
             request.setType(BiologicalDataItemResourceType.FILE);
         }
         switch (request.getType()) {
-            case GA4GH:
```

Comment:
Regarding GA4GH: are there some problems that prevent us for leaving this code? E.g. we can always replace genotype data with empty values as we are doing with NCBI?